app/models/feedback/steps/signup.rb

```diff
@@ -0,0 +1,29 @@
+module Feedback
+  module Steps
+    class Signup < ::GITWizard::Step
+      OPTIONS = [
+        OpenStruct.new(id: "1", value: "yes"),
+        OpenStruct.new(id: "0", value: "no"),
```

Comment:
```suggestion
        OpenStruct.new(id: "0", value: "No"),
```
Can we capitalise this @StackedPancakez ?